cmd/exo/stop.go

```diff
@@ -7,8 +7,11 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var stopNow = false
+
 func init() {
 	rootCmd.AddCommand(stopCmd)
+	stopCmd.Flags().BoolVar(&stopNow, "stop-now", false, "Attempts to stop the process immediately")
```

Comment:
I think that we should have "timeout" instead of "stop-now". Could be expressed in milliseconds, or we can maybe use the duration parsing logic - either from docker compose that I ported, or the builtin logic in Go.